instrumentation/events/lttng-module/net.h

```diff
@@ -8,7 +8,188 @@
 #include <linux/skbuff.h>
```

Comment:
about changelog and formatting output: the base (10, 16) used for each field does not match the output of wireshark. Please fix.